### DISCUSSION TITLE
P and E cores

### DISCUSSION THREAD

**Discussion by mgozdor2** [MODFLOW-ORG/modflow6#2020] [General]
Does the parallel version of MODFLOW use P cores, E cores, or a mixture of both?

**Comment by wpbonelli** [comment]
I think that will depend on the operating system and the MPI used to build parallel MF6, but @mjr-deltares is the expert on this

**Comment by mjr-deltares** [comment]
I think you are correct, @wpbonelli, MODFLOW relies on MPI for parallelization, both directly and through the PETSc library. MPI is not aware of hardware architecture on the caller's side. This question should be forwarded to the particular implementation you consider using, e.g. MPICH or OpenMPI. 

(Actually, there was an interesting discussion on the subject by the OpenMPI folks here: https://github.com/open-mpi/ompi/issues/11345 though that might be too much of a rabbit hole)
